usecases/configuration.go

```diff
@@ -143,32 +144,22 @@ func Configure(sys *components.System) ([]json.RawMessage, error) {
 		return rawBytes, err
```

Comment:
The above `os.ReadFile` is part of the previously mentioned race condition.

Reply:
In fact, why call `systemConfigFile, err := os.Open("systemconfig.json")` and then not use `systemConfigFile`? This is only done as a check to see if the file exists, which is... convoluted.

Reply:
Final note here, don't return both rawBytes and err. rawBytes can have have partial data that is useless, so just return nil for it.